src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -42,6 +80,26 @@
           scrollBehavior={scrollBehavior}
           isCentered
         >
+          <Button onClick={onOpen}>Open Modal</Button>
+
+          <Modal isOpen={isOpen} onClose={onClose}>
+            <ModalOverlay />
+            <ModalContent>
+              <ModalHeader>Modal Title</ModalHeader>
+              <ModalCloseButton />
+              <ModalBody>
+                <Lorem count={2} />
+              </ModalBody>
+
+              <ModalFooter>
```

Comment:
🚫 **[eslint]** <[react/jsx-no-undef](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-no-undef.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ModalFooter' is not defined.